server/src/types/resolvers/Channel/mutation.ts

```diff
@@ -229,3 +229,110 @@ export const leaveChannel = mutationField('leaveChannel', {
     });
   },
 });
+
+export const inviteUsersToChannel = mutationField('inviteUsersToChannel', {
+  type: 'Channel',
+
+  args: { channelId: stringArg({ nullable: false }), userIds: stringArg({ list: true, nullable: false }) },
+
+  description: `Adds some users into [public] channel.
+  `,
+
+  resolve: async (_, { channelId, userIds }, ctx) => {
+    const userId = getUserId(ctx);
+
+    const channels = await ctx.prisma.channel.findMany({
+      where: { id: channelId, deletedAt: null },
+      take: 1,
+    });
+    const channel = channels[0];
+
+    if (channel.channelType === 'private') {
+      throw new Error("You can't add some users to private channel.");
+    }
+
+    const memberships = await ctx.prisma.membership.findMany({
+      where: {
+        userId,
+        channelId: channel.id,
+      },
+      take: 1,
+    });
+    const membership = memberships[0];
+
+    if (membership.membershipType !== 'admin' && membership.membershipType !== 'owner') {
+      throw new Error('You should have admin or owner membership to add some users to the channel.');
+    }
+
+    for (const userId of userIds) {
+      const membership = await ctx.prisma.membership.findMany({
```

Comment:
`membership` => `memberships` since this is an array.

Reply:
Resolved in 3994d8e